src/components/ErrorBox/ErrorBox.tsx

```diff
@@ -1,27 +1,38 @@
-import React from 'react';
-import PropTypes from 'prop-types';
-import { renderServerValidationError } from '../../utils/FormValidationUtils';
+import React from "react";
+import { renderServerValidationError } from "../../utils/FormValidationUtils";
 
-const ErrorBox = ({ errorObject, className }) => {
+interface ErrorBoxProps {
+  className: string;
+  errorObject?: {
+    type: string;
+    details: any;
```

Comment:
Why can the type for `data` and `details` be `any`?

Reply:
They shouldn't be any. I'm not sure of the full extent of what `data` and `details` can be and the structure might change soon anyway. So for this round, they're all just `any`.